modules/models.py

```diff
@@ -889,6 +911,28 @@ class Game(BaseModel):
     size = ArrayField(SmallIntegerField, default=[0])
     is_mobile = BooleanField(default=True)
 
+    def as_json(self) -> dict[str, Any]:
```

Comment:
here too, `Dict` from `typing` not `dict`.

Reply:
This is a feature of Python 3.9, see [PEP 585](https://www.python.org/dev/peps/pep-0585/). If compatibility with 3.7 and 3.8 is required, `from __future__ import annotations` can be used.